geo/src/algorithm/interior_point.rs

```diff
@@ -0,0 +1,787 @@
+use std::cmp::Ordering;
+
+use crate::algorithm::{
+    bounding_rect::BoundingRect, centroid::Centroid, coords_iter::CoordsIter,
+    euclidean_distance::EuclideanDistance, line_intersection::LineIntersection,
+    lines_iter::LinesIter, relate::Relate,
+};
+use crate::geometry::*;
+use crate::sweep::*;
+use crate::GeoFloat;
+
+/// Calculation of interior points.
```

Comment:
```suggestion
/// Calculation of interior points.

```
Otherwise the docs render kind of funny without the newline:
<img width="448" alt="Screen Shot 2022-06-30 at 11 59 54 AM" src="https://user-images.githubusercontent.com/217057/176757151-8ffc5abd-e8ff-4859-a972-68f4aac6413c.png">